docs/src/main/asciidoc/security-guide.adoc

```diff
@@ -64,10 +64,6 @@ value. This cookie contains a expiry time as part of the encrypted value, so all
 clocks synchronised. At one minute intervals a new cookie will be generated with an updated expiry time if the session
 is in use.
 
-The following properties can be used to configure form based auth:
```

Comment:
Should we add a link to the form-auth section of all-doc-page?

Reply:
These properties are not generated in a smaller file?

Reply:
IIRC, they are not generated in a smaller file.

Reply:
Hmm, I don't understand. I remember having all of them in small files. Is it gone?

Reply:
Would it be possible to generate a file per config group too? Looks like we have quite a lot of docs where it makes sense.

Reply:
+1, I was leaning toward that path. I'll look at it.

Reply:
Why is it inexistant? I reviewed that guide last week and didn't see any inexistant file?

Reply:
The `FormAuthConfig` configuration is a config group, and we do not generate specific doc files for config groups. If it was previously there, maybe the file being referenced here was for a config root that is no longer there.

Reply:
Ah, that's possible.